railties/lib/rails/backtrace_cleaner.rb

```diff
@@ -10,17 +10,25 @@ class BacktraceCleaner < ActiveSupport::BacktraceCleaner # :nodoc:
 
     def initialize
       super
-      @root = "#{Rails.root}/"
+
+      from_prefix = "\tfrom "
```

Comment:
I'm not sure about this approach, it feels like a bandaid over the real problem which is the backtrace format is not stable. Maybe instead of adding this new filter we should try to modify the existing code to do more specifically what we want here: not showing backtrace lines from outside of the app, only show relative app paths instead of absolute paths, and don't show the RENDER_TEMPLATE_PATTERN.

If I were to write this class from scratch my thinking would be to do something like this:
- modify the silencer to check that a backtrace line contains the app root, and run that first
- then run a filter that replaces the absolute path with a relative path and removes the RENDER_TEMPLATE_PATTERN

Since the class is `nodoc` we may be able to just do this, but I'm also not a committer so it may be a good idea to get another opinion.